packages/font/src/google/get-proxy-agent.ts

```diff
@@ -0,0 +1,22 @@
+// @ts-ignore
+import { warnOnce } from 'next/dist/shared/lib/utils/warn-once'
+import { Agent } from 'https'
+
+/**
+ * If the https_proxy environment variables is set, try to import and return a proxy agent.
+ * Prints a warning if the proxy agent couldn't be imported.
+ */
+export function getProxyAgent(): Agent | undefined {
+  const httpsProxy = process.env['https_proxy'] || process.env['HTTPS_PROXY']
+  if (httpsProxy) {
+    try {
+      const HttpsProxyAgent = require('https-proxy-agent')
+      return new HttpsProxyAgent(httpsProxy)
```

Comment:
Should we pre-compile this and re-use it elsewhere we are fetching external resources?

Reply:
Added https-proxy-agent and http-proxy-agent to